editor.planx.uk/src/@planx/components/Pay/Public/Pay.tsx

```diff
@@ -76,6 +75,11 @@ function Component(props: Props) {
 
   const metadata = [...(props.govPayMetadata || []), ...defaultMetadata];
 
+  const errorMessage =
+    teamSlug !== "demo"
+      ? "GOV.UK Pay is not enabled for this local authority"
+      : "GOV.UK Pay is not enabled for the Demo team";
```

Comment:
nit: does "Demo team" have meaning to outsiders? Would something like "GOV.UK Pay is not enabled for demonstrations" or "GOV.UK Pay is not enabled in this trial workspace" be more generic and less internal-jargony?

Reply:
I guess it depends what you mean by outsiders, I'm imagining it is the `demoUser` testing their services themselves or testing them with colleagues because they shouldn't be testing this with the actual public. 

For these people, they would access their flows and create new flows through the "Demo" team, so they would have a notion of what the `Demo` team is. 

Demonstrations or "trial workspace" might bring new terminology here in amongst being a demo user inside the demo team, so I'd rather keep the demo team wording.  

For instance, if you were to do a demonstration for someone, you would probably want to show planning constraints, successful GOV.UK Pay stuff etc... so it's different to what the `Demo` team is used for, which is a space for new councils to experiment and test the platform? 

I agree that the wording should change though... I'll have a think and maybe ask the Content Team or Sylvia